src/frontend/tsx/GamePanel.tsx

```diff
@@ -9,70 +9,71 @@ import { Ship, Side, GameID, SAN, GameInfo, ActiveGameInfo } from '../ts/types/u
 export function GamePanel () {
   const { urbit, displayGame, setDisplayGame, practiceBoard, setPracticeBoard, displayIndex, setDisplayIndex } = useChessStore()
   const hasGame: boolean = (displayGame !== null)
+  const activeGame: boolean = hasGame && !(displayGame.archived)
```

Comment:
`hasActiveGame` would be a better name. Also, I just realized that we don't need `hasGame` at all: it's a leftover from when the `GamePanel` and `PracticePanel` weren't different panels. We can remove all usages of `hasGame`.

Reply:
I didn't even realize there was a `PracticePanel`. That's nice.